ipa.tf

```diff
@@ -48,6 +48,11 @@ locals {
   ] : []
   storage_spec = var.include_fsx == true ? local.fsx_values : local.efs_values
   acm_ipa_values = var.use_acm == true ? (<<EOT
+runtime-scanner:
+  enabled: ${replace(lower(var.aws_account), "indico", "") == lower(var.aws_account) ? "false" : "true"}
+  authentication:
+    ingressUser: monitoring
+    ingressPassword: ${random_password.monitoring-password.result}
```

Comment:
Do the runtime scanner values need to be here as well? Looks like they are being set in the Helm values at line 651 as well